schema/v1.1.0/common.yaml

```diff
@@ -270,28 +285,49 @@ slots:
     description: Reference to the run this tiltseries is a part of
   tiltseries_acceleration_voltage:
     range: integer
+    minimum_value: 0
+    required: true
     description: Electron Microscope Accelerator voltage in volts
     unit:
       descriptive_name: volts
       symbol: V
   tiltseries_aligned_tiltseries_binning:
     range: integer
+    # TODO: uncomment after any_of is fixed https://github.com/linkml/linkml/issues/1521
+    # any_of:
+    #   - range: integer
+    #   - range: IntegerFormattedString
     description: Binning factor of the aligned tilt series
+    ifabsent: 'int(1)'
```

Comment:
I'm curious about this. Why do we need to specify `'int(1)'` instead of just `1` ?

Reply:
To prevent code injection I believe, see: https://github.com/linkml/linkml/blob/48241d48829a4311d3a9c96779cb241d7b829e56/linkml/utils/ifabsent_functions.py#L62-L122